cli/internal/kubecmd/kubecmd.go

```diff
@@ -124,17 +124,13 @@ func (k *KubeCmd) UpgradeNodeVersion(ctx context.Context, conf *config.Config, f
 	nodeVersion.Spec.ImageReference = imageReference
 	nodeVersion.Spec.ImageVersion = imageVersion.Version()
 
+	// TODO(elchead): why?
```

Comment:
Because we don't to force a user to upgrade their Kubernetes version with each release.
If we publish a new Constellation version which ships with Kubernetes `v1.27.4`, and a user has their cluster running on Kubernetes `v1.27.2`, they should be able to run `constellation upgrade apply` without necessarily increasing their Kubernetes patch version,